apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -129,7 +133,7 @@ class CourseScript extends Component {
             color={Button.ButtonColor.gray}
             className="uitest-go-to-unit-button"
           />
-          {experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
+          {isAssigned && experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
```

Comment:
is it ok for this to show up for teachers? they might have been assigned by their facilitator during PD.

Reply:
Yes, we want it to show up for teachers too. If a teacher was assigned the script as a student in a section (say at PD) or more likely if they assigned the unit to one of their owned sections then they will see the assigned checkmark.